configs/sites/gaea-c5/compilers.yaml

```diff
@@ -0,0 +1,27 @@
+compilers:
+- compiler:
+    spec: intel@2022.0.2
+    paths:
+      cc: cc
+      cxx: CC
+      f77: ftn
+      fc: ftn
+    flags: {}
+    operating_system: sles15
+    target: any
+    modules:
+    - PrgEnv-intel/8.3.3
+    - intel/2022.0.2
+    - craype/2.7.15
+    - libfabric/1.11.0.4.128
```

Comment:
Is there a reason this needs to be here and not just under cray-mpich in packages.yaml? I assume this was the fix for the libfabric problem but I'm at a loss as to why my build wasn't working... Maybe something to do with module load order?

Reply:
I honestly can't tell you the reason, but if I put it here than the build works. It's also in the cray-mpich package, but specifying it twice doesn't hurt.